alpaca_trade_api/stream2.py

```diff
@@ -227,6 +231,7 @@ def __init__(
         self._data_stream = _data_stream
         self._debug = debug
         self._raw_data = raw_data
+        self.stop_stream_queue = queue.Queue()
```

Comment:
[minor] should be private: `_stop_stream_queue`

Reply:
done